app/models/kudo.rb

```diff
@@ -1,6 +1,7 @@
 class Kudo < ApplicationRecord
   belongs_to :receiver, class_name: 'Employee'
   belongs_to :giver, class_name: 'Employee'
+  belongs_to :company_value
 
   validates :title, presence: true
   validates :content, presence: true
```

Comment:
you should validate the presence of company_value as well, especially when you have `not_null` in db here

Reply:
Added CompanyValue presence validation as suggested